linkcheck/lc_cgi.py

```diff
@@ -54,7 +55,7 @@ def application(environ, start_response):
         request_body = environ['wsgi.input'].read(request_body_size)
     else:
         request_body = environ['wsgi.input'].read()
-    form = urlparse.parse_qs(request_body)
+    form = urlparse.parse_qs(request_body.decode())
```

Comment:
Maybe `.decode(self.HTML_ENCODING)`?

This looks like a wsgi script that is meant to accept data from real users, so we should not crash in the presence of non-ASCII data.

Reply:
Changed this (HTML_ENCODING it is not part of a class).